docs/pages/includes/kubernetes-access/rbac.mdx

```diff
@@ -51,4 +51,24 @@ $ tctl create -f kube-user.yaml
 $ tctl create -f user.yaml
 ```
 
-Log out of Teleport and log in again.
\ No newline at end of file
+Create a ClusterRoleBinding for the "basic-viewers" group:
+
+```code
+$ cat <<EOF | kubectl apply -f -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: basic-viewers-bind
+subjects:
+- kind: Group
+  name: basic-viewers # corresponds to the kubernetes_group we assigned our "kube-user" role above
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name: view # A default ClusterRole with read-only access: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles
```

Comment:
I think this comment would look a bit nicer as multiple lines placed above the `name` field.